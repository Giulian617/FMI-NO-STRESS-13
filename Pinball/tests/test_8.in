93 1
71 59 3
2 71 1
8 2 5
10 71 9
48 2 3
90 71 2
51 90 7
25 71 6
47 10 1
58 8 2
3 51 9
46 8 5
66 46 4
93 47 6
40 47 8
81 71 5
62 25 4
11 8 8
87 47 4
92 62 10
86 71 6
74 87 5
82 66 3
30 58 1
16 82 9
53 16 9
88 92 9
69 58 7
29 92 10
67 46 4
57 40 2
63 93 4
75 82 6
34 67 3
56 63 6
89 88 3
22 57 4
42 63 8
45 69 8
91 67 7
83 89 5
39 42 7
20 45 2
78 69 8
79 45 10
33 83 1
9 45 1
80 33 5
6 22 5
35 57 6
32 79 10
50 83 1
12 20 1
44 91 4
55 22 10
54 33 6
65 20 5
5 42 10
41 83 7
77 5 6
70 83 8
60 65 4
4 44 4
64 5 7
36 35 9
13 4 6
14 65 5
26 41 8
68 32 4
73 54 7
19 73 5
49 36 7
61 13 10
27 61 5
84 41 2
31 19 6
24 27 2
23 77 8
85 64 4
76 64 4
21 23 7
18 27 5
15 76 1
37 14 2
28 23 10
17 84 2
1 23 1
52 26 2
7 85 4
72 76 3
38 49 3
43 84 1
