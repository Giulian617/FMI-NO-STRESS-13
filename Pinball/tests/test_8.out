987769796
889409761
737441203
305299224
393932211
26079480
617407112
581950478
62076516
4035292
244914841
740291461
627697895
204503953
520330847
10330799
670167053
159829450
145834114
309112659
284610372
566152618
15901794
890343042
537611109
14657955
533462675
716242272
285364733
139717665
50231688
858322016
581126574
961391125
533782712
450388768
571987615
770291242
